fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/RelOpValue.java

```diff
@@ -654,6 +654,14 @@ private enum BinaryPhysicalOperator {
         NEQ_BYU(Comparisons.Type.NOT_EQUALS, Type.TypeCode.BYTES, Type.TypeCode.UNKNOWN, (l, r) -> null),
         NEQ_BYBY(Comparisons.Type.NOT_EQUALS, Type.TypeCode.BYTES, Type.TypeCode.BYTES, (l, r) -> Comparisons.evalComparison(Comparisons.Type.NOT_EQUALS, l, r)),
         NEQ_UBY(Comparisons.Type.NOT_EQUALS, Type.TypeCode.UNKNOWN, Type.TypeCode.BYTES, (l, r) -> null),
+        LT_BYU(Comparisons.Type.LESS_THAN, Type.TypeCode.BYTES, Type.TypeCode.UNKNOWN, (l, r) -> null),
```

Comment:
I know this is just following the same patterns but we shouldn't have implicit promotions for anything with `Type.TypeCode.UNKNOWN` in it. That's just plain wrong. `UNKNOWN` is reserved for stuff created by the heuristic planner, but that code path should never really use this `Value`.

Reply:
So those should all be removed? Will something break?

Reply:
They should all be removed, but it is possible that something breaks downstream. Can you log an issue?

Reply:
https://github.com/FoundationDB/fdb-record-layer/issues/2915